traffic_monitor/cache/stats_over_http.json_new

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
From Weasel:

```
Error                                 Unknown! traffic_monitor/cache/stats_over_http.json_new
```

Is `stats_over_http.json_new` needed?

Reply:
nope, submitted by mistake

Reply:
gotcha